testbed/eks/package.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
I'm not seeing the value of this living inside and eks folder. Thoughts on just moving this to the top level of the testbed?